Chinese/大话西游.md

```diff
@@ -1,4 +1,5 @@
-在1994年大话西游之月光宝盒的电影中，主角周星驰饰演的孙悟空在500年后转世成为：
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.